hacep/src/main/java/org/kie/hacep/core/infra/election/LeaderElectionImpl.java

```diff
@@ -80,7 +85,7 @@ public void addCallbacks(List<LeadershipCallback> callbacks) {
         this.callbacks.addAll(callbacks);
     }
 
-    private void refreshStatus() {
+    public void refreshStatus() {
```

Comment:
What is the reason to change many methods from `private` to `public`? If it is for testing purpose I would suggest again `default` or `protected`